### PR TITLE
Changelog for transport-23

### PR DESCRIPTION
**Pull request by martinthomson**
Someone already did TLS for me.

I put the major wire-format change first.  ekr likes the idea of marking such things as "special", but I don't subscribe to that theory.  This is all important.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -5935,6 +5935,38 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
...
+- Allow smaller stateless resets and recommend a smaller minimum on packets
+  that might trigger a stateless reset (#2770, #2869, #2927)
+- Provide guidance around the interface to QUIC as used by application protocols
+  (#2805, #2857)
+- Frames other than STREAM can cause STREAM_LIMIT_ERROR (#2825, #2826)
+- Tighter rules about processing of rejected 0-RTT packets (#2829, #2840, #2841)
+- Explanation of the effect of Retry on 0-RTT packets (#2842, #2852)
+- Cryptographic handshake needs to provide server transport parameter encryption
+  (#2920, #2921)
+
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]: If #2916 is merged,
- Moved guidance around ACK generation from recovery draft to here (#2916).

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -5935,6 +5935,38 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-quic-transport-22
+
+- NEW_CONNECTION_ID includes a request to retire old connection IDs (#2645,
```
Comment (marten-seemann) [inline draft-ietf-quic-transport.md]: That was already part of draft-22.

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -5935,6 +5935,38 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
...
+  #2769)
+- Rules for preventing correlation by connection ID tightened (#2084, #2929)
+- Clarified use of CONNECTION_CLOSE in Handshake packets (#2151, #2541, #2688)
+- Discourage regressions of largest acknowledged in ACK (#2205, #2752)
+- Improved robusness of validation process for ECN counts (#2534, #2752)
+- Introduce concept of handshake confirmation (#2267, #2673)
```
Comment (marten-seemann) [inline draft-ietf-quic-transport.md]: This as well.

**Comment by martinthomson** [comment]
Thanks @marten-seemann, I mistyped the date when I searched and didn't check for duplicates on that set.

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (janaiyengar) [reply draft-ietf-quic-transport.md]: #2916 is now merged.

**Pull request merged**